### ISSUE TITLE
[Feature] <write a small description here>Cart 

### ISSUE THREAD

**Issue by kushwahaPrashant24** [ShwetKhatri2001/AR-Webstore#192]
### Description

Add Cart Section Feature

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines]
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

**Comment by kushwahaPrashant24** [comment]
I want to create a cart section so that we can add our selected item in that. Please assign me this task

**Comment by ShwetKhatri2001** [comment]
Sure, Go ahead.

**Comment by kushwahaPrashant24** [comment]
Yes I am contributing This as a part of Kwoc 2023  open-source program.

**Comment by ShwetKhatri2001** [comment]
Any updates or issues @kushwahaPrashant24 ?

**Comment by kushwahaPrashant24** [comment]
I am working on it and do not have any issue

**Comment by ShwetKhatri2001** [comment]
Please try to complete it in the next 3 days.

**Comment by ShwetKhatri2001** [comment]
Let's give other contributors a chance now. This issue is open to contributors.

**Comment by FlareMozZ** [comment]
I can work on adding a cart feature to your website. Please assign me the issue

**Comment by ShwetKhatri2001** [comment]
Assigning you @FlareMozZ . If you're still interested then start working on it.

**Comment by FlareMozZ** [comment]
yeah, I'm on it, looking forward for the learning curve

**Comment by ShwetKhatri2001** [comment]
@FlareMozZ Reminding you SWOC contribution period is ending on **`28 Feb`** if you're contributing as part of it.

**Comment by ayann07** [comment]
Since I have successfully implemented the wishlist section recently, should I give a try in building this?

**Comment by FlareMozZ** [comment]
@ShwetKhatri2001 thank you for the remainder, I am indeed contributing under SWOC, but I wont be able to deliver complete feature yet due to exams.

**Comment by ayann07** [comment]
@ShwetKhatri2001 can I contribute to this? Already I am doing this for my experience, I will not be doing this under any open source program.

**Comment by ShwetKhatri2001** [comment]
Okay, go ahead.
